tasks/sanity-optional-label-check.yaml

```diff
@@ -0,0 +1,19 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: sanity-optional-label-check
+spec:
+  params:
+    - name: sourceImage
+      description: the fully qualified image name
+  workspaces:
+    - name: workspace
+
+  steps:
+    - name: optional-label-check
+      image: quay.io/redhat-appstudio/hacbs-test:stable
+      workingDir: $(workspaces.workspace.path)/hacbs/$(context.task.name)
+      script: |
+        skopeo inspect --no-tags docker://$(inputs.params.sourceImage) > optional_label_image_inspect.json
+        /usr/bin/conftest test optional_label_image_inspect.json --policy /project/image/optional-labels.rego --output=json
```

Comment:
the result:
```
step-optional-label-check
+ skopeo inspect --no-tags docker://image-registry.openshift-image-registry.svc:5000/project/single-container-app:37daed8
+ /usr/bin/conftest test optional_label_image_inspect.json --policy /project/image/optional-labels.rego --output=json
[
	{
		"filename": "optional_label_image_inspect.json",
		"namespace": "main",
		"successes": 0
	}
]
```
Having no idea if the image is ok or not.

Probably there should be `--no-fail` as in sanity-label-check. Result should be stored as result -> HACBS_TEST_OUTPUT